MekHQ/src/mekhq/campaign/Campaign.java

```diff
@@ -2116,6 +2116,21 @@ public List<Person> getActivePersonnel() {
                 .collect(Collectors.toList());
     }
 
+    /**
+     * Retrieves a filtered list of personnel who have at least one combat profession.
+     * <p>
+     * This method filters the list of all personnel to include only those whose primary
+     * or secondary role is designated as a combat role.
+     * </p>
+     *
+     * @return a {@link List} of {@link Person} objects representing combat-capable personnel
+     */
+    public List<Person> getActiveCombatPersonnel() {
+        return getPersonnel().stream()
```

Comment:
Should this be getting all personnel or just active personnel?

Reply:
Fudge, well spotted

Reply:
Fixed - thanks again :)